ci-operator/step-registry/hypershift/kubevirt/install/hypershift-kubevirt-install-commands.sh

```diff
@@ -2,7 +2,19 @@
 
 set -ex
 
-CNV_PRERELEASE_VERSION=${CNV_PRERELEASE_VERSION:-}
+function calculate_cnv_version() {
+    local ocp_version=$(oc get clusterversion --no-headers | awk '{ print $2 }' | awk -F "." '{print $1"."$2}')
```

Comment:
i think something like this will be more clean/elegant than parsing cli output with awk:
```
oc get clusterversion version -o jsonpath='{.status.desired.version}' | awk -F "." '{print $1"."$2}'
```